src/elements/content-preview/ContentPreview.js

```diff
@@ -1153,7 +1156,8 @@ class ContentPreview extends PureComponent<Props, State> {
                             ) : (
                                 <div className="bcpr-loading-wrapper">
                                     <PreviewLoading
-                                        isLoading={!isFileError}
+                                        errorCode={errorCode}
```

Comment:
Should we create a separate component for this blocked by access policy case instead of re-using `PreviewLoading`? Seems incorrect to use a loading state component to display the final error

Reply:
This component isn't loading state, it was always error state. Makeloadable is loading state.

Reply:
Gotcha, was thrown off by all the loading classnames

Reply:
Probably can eventually rename